Userland/Games/GameOfLife/Board.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2021, Andres Crucitti <dasc495@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Vector.h>
+#include <LibGfx/Point.h>
+#include <stdio.h>
+
+class Board {
+public:
+    Board(size_t columns, size_t rows);
```

Comment:
The conventional order throughout this entire codebase is `row, column` instead of `column, row`